threema-connector-webtest/src_test/threema/connector/webtest/MessageSingleRecipientTest.java

```diff
@@ -7,21 +7,33 @@
 import static com.codeborne.selenide.Selenide.$$;
 import static com.codeborne.selenide.Selenide.open;
 import static org.assertj.core.api.Assertions.assertThat;
-import org.junit.jupiter.api.Disabled;
+
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Test;
 import org.openqa.selenium.By;
+
 import com.axonivy.ivy.webtest.IvyWebTest;
 import com.axonivy.ivy.webtest.engine.EngineUrl;
+import com.axonivy.ivy.webtest.engine.WebAppFixture;
 import com.axonivy.ivy.webtest.primeui.PrimeUi;
 import com.codeborne.selenide.ElementsCollection;
 
+import ch.ivyteam.threema.mocks.ThreemaServiceMock;
+
 @IvyWebTest
 public class MessageSingleRecipientTest {
 
+  @AfterEach
+  public void cleanFixture(WebAppFixture fixture) {
+    fixture.resetConfig("RestClients.ThreemaGateway.Url");
+  }
+
+
   @Test
-  @Disabled
-  public void sendMessage() {
-    open(EngineUrl.createProcessUrl("threema-connector-demo/18B22F69680901D3/start.ivp"));
+  public void sendMessage(WebAppFixture fixture) {
+    String mockUrl = ThreemaServiceMock.URI.replaceAll("\\{", "%7B").replaceAll("\\}", "%7D").replaceAll("/", "%2F");
+    fixture.config("RestClients.ThreemaGateway.Url", mockUrl);
```

Comment:
I don't see anything obviously wrong in theses tests; can you reproduce that stuff locally by running `mvn clean verify` in the threema repo?

Reply:
Running it with the same arguments as the CI-Build works 
`mvn clean verify -Dmaven.test.failure.ignore=true -Divy.engine.version=10.0.14`

But running it just as `mvn clean verify` fails with:
```
[ERROR]   GetReceiverInfoTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
[ERROR]   MessageEncryptionTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
[ERROR]   MultipleRecipientsTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
[ERROR]   SendMessageTest » ExceptionInInitializer
[ERROR]   SingleRecipientTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
```

Reply:
the normal clean verify may fails due to cached ivy engines in your local maven repo: you can purge them like this `rm -rf $HOME/.m2/repository/.cache/ivy`